tools/intersmash-tools-provisioners/src/main/resources/crds/keycloak.org_keycloakclients_crd.yaml

```diff
@@ -665,7 +665,7 @@ spec:
                             items:
                               type: string
                             type: array
-                          description: Map client => []role
+                          description: Map client =&gt; []role
```

Comment:
is it really required to changes this in yaml files? Aren't crd generated automatically?

Reply:
The Javadoc Maven plugin would break the build treating those as errors, if we don't fix them.
But there might be an option for this to be skipped, or at least I can check whether it exists.

Reply:
Yes, here it is description but I worry that it might be a problem for some real value

Reply:
Hi @simkam - it looks like I managed to keep the original contents and to skip the Javadoc verification process, see https://github.com/Intersmash/intersmash/pull/119/commits/6af29d314ea14572a9197802cf628017873838af